src/main.py

```diff
@@ -119,5 +135,176 @@ async def search_items_API(
         return None
 
 
+@app.get("/analysis/varietyCount/all/{item_name}", response_model=List[analysisVarietyCountJson])
+async def items_variety_count_analysis_API(
+    item_name: str,
+    order_by_col: Optional[str] = None,
+    reverse: Optional[bool] = False
+):
+    ''' Wrapper API to fetch the count of number of varieties for a particular item found  in 
+    AMAZON, WALMART, TARGET, COSTCO, BESTBUY, EBAY query results
+    Parameters
+    ----------
+    item_name: string of item to be searched
+
+    Returns
+    ----------
+    itemListJson: JSON List
+        list of count of varieties of the item across all websites as JSON List
+    '''
+
+    # building argument
+    args = {
+        'search': item_name,
+        'sort': 'pr' if order_by_col == 'price' else 'pr',  # placeholder TDB
```

Comment:
what is the placeholder here?

Reply:
Where will this data be shown? Will it be displayed on the results page?

Reply:
This can be used in future to provide sorting by price as a parameter from the UI. Currently it is just left as a place holder with the default sorting done in ascending order of price